src/components/shared/Button/Close.js

```diff
@@ -14,13 +14,13 @@ type Props = {
 
 function CloseButton({ handleClick, buttonClass, tooltip }: Props) {
   return (
-    <div
+    <button
```

Comment:
This breaks close buttons (internal 'x' svg image now seems to have width of 0). You can also see how close buttons within source tabs are also broken. I think you could use implementation of toggle-button-end and toggle-button-start buttons (in the same directory) as a good example of how to make real buttons out of "div" style buttons.

Reply:
This seems fine now.